src/java.base/share/classes/java/lang/invoke/MethodHandleProxies.java

```diff
@@ -202,7 +202,8 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                     if (isObjectMethod(method))
                         return callObjectMethod(proxy, method, args);
                     if (isDefaultMethod(method)) {
-                        return callDefaultMethod(defaultMethodMap, proxy, intfc, method, args);
+                        // no additional access check is performed
+                        return JLRA.invokeDefault(proxy, method, null, args);
```

Comment:
Isn't the argument order `..., args, caller`?

Reply:
Such edit was an accident.  Fixed.